toolchain/config.bzl.tpl

```diff
@@ -105,7 +105,7 @@ def _impl(ctx):
                             "-Wl,-z,relro,-z,now",
                             "-no-canonical-prefixes",
                             "-pass-exit-codes",
-                            "-lstdc++",
+                            "-l:libstdc++.a",
```

Comment:
should the explanation from the commit message end up in some comment or docstring in the code?